<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 85, 139);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Library Management System</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bookid">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border : 2px solid rgb(255, 255, 255);
border-radius: 10px;
background-color:rgb(136, 154, 188);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Book-No (1001)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearbut">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>500</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color:rgb(144, 177, 255);</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Adding New Book</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bname">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border : 2px solid rgb(255, 255, 255);
border-radius: 10px;
background-color:rgb(136, 154, 188);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Book Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="authname">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border : 2px solid rgb(255, 255, 255);
border-radius: 10px;
background-color:rgb(136, 154, 188);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Author Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="booked">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border : 2px solid rgb(255, 255, 255);
border-radius: 10px;
background-color:rgb(136, 154, 188);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter Book Edition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBbut">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color:rgb(144, 177, 255);</string>
    </property>
    <property name="text">
     <string>Add Book</string>
    </property>
   </widget>
   <widget class="QLabel" name="showtext">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dashB">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>570</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color:rgb(144, 177, 255);</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>381</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(56, 85, 139);
border-radius: 20px;
border : 2px solid rgb(144, 177, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>bookid</zorder>
   <zorder>clearbut</zorder>
   <zorder>label_2</zorder>
   <zorder>bname</zorder>
   <zorder>authname</zorder>
   <zorder>booked</zorder>
   <zorder>addBbut</zorder>
   <zorder>showtext</zorder>
   <zorder>dashB</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
